### PR TITLE
Base branch: main [do not merge]

### PR DESCRIPTION
**Pull request by mxdvl**
## What does this change?

Do not merge. This allows comparison between branches `main` and `master`.

## Does this change need to be reproduced in dotcom-rendering ?

- [ ] No
- [X] Yes (please indicate your plans for DCR Implementation)

### Tested

- [ ] Locally
- [ ] On CODE (optional)

### CODE REVIEW

File: docs/03-dev-howtos/16-working-with-amp.md [hunk 1]
```diff
@@ -64,5 +64,5 @@ This change was made to improve the experience of developing and styling amp pag
 
 ## Ads in Amp
 
-* [Prebid](https://prebid.org/) in Amp uses a Prebid server and is documented in a separate repo [here](https://github.com/guardian/prebid-server/tree/master/amp).
+* [Prebid](https://prebid.org/) in Amp uses a Prebid server and is documented in a separate repo [here](https://github.com/guardian/prebid-server/tree/main/amp).
```
Comment (amyhughes) [inline docs/03-dev-howtos/16-working-with-amp.md:67]: Perhaps this should be left out until the prebid-server repo is switched to main?

File: docs/99-archives/incidents/2013-03-13a.md [hunk 2]
```diff
@@ -6,7 +6,7 @@
 
 ## Summary
 
-At 11am 13 March the merge of [scala 2.10 & play 2.1](https://github.com/guardian/frontend/pull/512) was made to master. This was successfully built and deployed around lunchtime that same day.
+At 11am 13 March the merge of [scala 2.10 & play 2.1](https://github.com/guardian/frontend/pull/512) was made to `main`. This was successfully built and deployed around lunchtime that same day.
```
Comment (amyhughes) [inline docs/99-archives/incidents/2013-03-13a.md:9]: I'm not sure it makes sense to change an incident report?

File: docs/99-archives/incidents/2013-06-25.md [hunk 3]
```diff
@@ -15,11 +15,11 @@ articles etc.) remained accessible.
 
 ## Cause
 
-On Friday 21 June @ironsidevsquincy made a pull request to master for some improvements to the AB testing framework.
+On Friday 21 June @ironsidevsquincy made a pull request to `main` for some improvements to the AB testing framework.
```
Comment (amyhughes) [inline docs/99-archives/incidents/2013-06-25.md:18]: Again, it wasn't 'main' at the time of the incident

File: docs/99-archives/incidents/2013-07-11.md [hunk 4]
```diff
@@ -23,7 +23,7 @@ We have 3 layers of healthchecks, instance ELB, router ELB and Fastly. Always re
 When you are not immediately sure what exactly is wrong, rolling back the entire stack feels like a big job
 (fortunately it did not come to that, but there was a moment of worry)
 
-Another developer merged to MASTER while this was ongoing and triggered a round of builds and automated deploys that
+Another developer merged to `main` while this was ongoing and triggered a round of builds and automated deploys that
```
Comment (amyhughes) [inline docs/99-archives/incidents/2013-07-11.md:26]: Again

**Comment by amyhughes** [comment]
Looks good, other than a couple of comments!

File: docs/99-archives/incidents/2013-07-01.md [hunk 5]
```diff
@@ -22,8 +22,8 @@ During the incident there was a drop of about 20% in traffic (see attached Ophan
 - Trim all url paths supplied by the content API to avoid the underlying issue in the future. GK fixed this with this [commit](https://github.com/guardian/frontend/commit/bbad0a7eaf1db7e37b0fc5bf7d611d9a378875d8)
 
 ## Appendix
-![Exception](https://raw.github.com/guardian/frontend/master/docs/incidents/images/2013-07-01-exception.png)
-![Ophan](https://raw.github.com/guardian/frontend/master/docs/incidents/images/2013-07-01-ophan.png)
+![Exception](https://raw.github.com/guardian/frontend/main/docs/incidents/images/2013-07-01-exception.png)
+![Ophan](https://raw.github.com/guardian/frontend/main/docs/incidents/images/2013-07-01-ophan.png)
```
Comment (mxdvl) [inline docs/99-archives/incidents/2013-07-01.md:26]:
```suggestion
![Exception](https://raw.github.com/guardian/frontend/main/docs/99-archive/incidents/images/2013-07-01-exception.png)
![Ophan](https://raw.github.com/guardian/frontend/main/docs/99-archive/incidents/images/2013-07-01-ophan.png)
```

Comment (mxdvl) [inline docs/99-archives/incidents/2013-07-01.md:26]: These have actually moved.

```suggestion
![Exception](https://raw.github.com/guardian/frontend/main/docs/99-archives/incidents/images/2013-07-01-exception.png)
![Ophan](https://raw.github.com/guardian/frontend/main/docs/99-archives/incidents/images/2013-07-01-ophan.png)
```

**Comment by mxdvl** [comment]
We should not push directly to `main`!

See #22979 for the actual changes.

**Pull request closed without merging**